src/components/Error.tsx

```diff
@@ -1,4 +1,15 @@
-const Error = ({ title, link, children }) => {
+import React, { ReactNode } from 'react';
+
+interface ErrorProps {
+  title: string;
+  link?: {
+    url: string;
+    name: string;
+  };
+  children: ReactNode;
+}
+
+const Error: React.FC<ErrorProps> = ({ title, link, children }) => {
   return (
```

Comment:
I would use `PropsWithChildren<T>` instead of defining children itself and flatten out the props.

Reply:
I don't think there is any issue with it, there could be multiple implementations, and everybody has their own way of doing it.

Reply:
It was purely a suggestion. 
As additional information, flatting the props would help react at the diffing phase as you would only use primitives instead of referential types which could help later on to optimise the performance of your application if it is really needed.